test/payment/PaymentSplitter.test.js

```diff
@@ -4,6 +4,9 @@ const { sendEther } = require('./../helpers/sendTransaction');
 const { ether } = require('../helpers/ether');
 const { ZERO_ADDRESS } = require('./../helpers/constants');
 
+const BigNumber = web3.BigNumber;
+require('../helpers/setup').shouldWithBigNumber(BigNumber);
```

Comment:
I may not have been clear enough, what I meant is that we don't want the user to provide a `BigNumber` object, but rather to use our own _inside_ the `setup` helper.

These two lines should just be:
```
const { BigNumber } = require('../helpers/setup');
```

Reply:
Got it now